diff --git a/libexec/alluxio-config.sh b/libexec/alluxio-config.sh
index 9af01b9ca3..8118ce9f0f 100755
--- a/libexec/alluxio-config.sh
+++ b/libexec/alluxio-config.sh
@@ -29,8 +29,8 @@ this="${config_bin}/${script}"
 # set system installation locations.
 VERSION=2.9.0-SNAPSHOT
 ALLUXIO_HOME=$(dirname $(dirname "${this}"))
-ALLUXIO_ASSEMBLY_CLIENT_JAR="${ALLUXIO_HOME}/assembly/client/target/alluxio-assembly-client-${VERSION}-jar-with-dependencies.jar"
-ALLUXIO_ASSEMBLY_SERVER_JAR="${ALLUXIO_HOME}/assembly/server/target/alluxio-assembly-server-${VERSION}-jar-with-dependencies.jar"
+ALLUXIO_ASSEMBLY_CLIENT_JAR="${ALLUXIO_HOME}/assembly/alluxio-client-${VERSION}.jar"
+ALLUXIO_ASSEMBLY_SERVER_JAR="${ALLUXIO_HOME}/assembly/alluxio-server-${VERSION}.jar"
 ALLUXIO_CONF_DIR="${ALLUXIO_CONF_DIR:-${ALLUXIO_HOME}/conf}"
 ALLUXIO_LOGS_DIR="${ALLUXIO_LOGS_DIR:-${ALLUXIO_HOME}/logs}"
 ALLUXIO_USER_LOGS_DIR="${ALLUXIO_USER_LOGS_DIR:-${ALLUXIO_LOGS_DIR}/user}"
